gatsby-node.esm.js

```diff
@@ -300,6 +305,13 @@ exports.createPages = async ({ actions, graphql }) => {
             data: pivotedData,
           },
         })
+        /*createPage({
+          path: `${path}/comingsoon/`,
+          component: ComingSoon,
+          context: {
+            data: pivotedData,
+          },
+        })*/ // Currently commented out as we don't want to generate these pages yet. TODO: @jtan re-enable.
         response.data.allReferrersYaml.edges.forEach(({ node }) => {
```

Comment:
I think missing this AC?: "The same page is also shown for all vanity referrer pages." The value of that is we can give out referrer links that work even prior to launching.

Reply:
Yep, I think it is missing that